training/Exercise04_Bayesian_Inverse_Problems.ipynb

```diff
@@ -347,7 +392,7 @@
     "\n",
     "where we use the notation $L(\\mathbf{x} \\mid \\mathbf{y}) := p(\\mathbf{y} \\mid \\mathbf{x})$ for the likelihood function to emphasize that it is a function of $\\mathbf{x}$ and not on $\\mathbf{y}$.\n",
     "\n",
-    "For the specific case of having observed $\\mathbf{y}_\\mathrm{data}$, we write posterior distribution as $p(\\mathbf{x} \\mid \\mathbf{y}=\\mathbf{y}_\\mathrm{data})$.\n",
+    "For the specific case of having observed $\\mathbf{y}_\\mathrm{data}$, we write posterior distribution as $p(\\mathbf{x} \\mid \\mathbf{y}=\\mathbf{y}_\\mathrm{data})$ and $L(\\mathbf{x} \\mid \\mathbf{y}=\\mathbf{y}_\\mathrm{data})$.\n",
```

Comment:
I found the added part "and $L(\\mathbf{x} \\mid \\mathbf{y}=\\mathbf{y}_\\mathrm{data})$" a bit confusing but maybe I am missing something.
This is related to my comment above about the distinction between the data distribution and the likelihood. I tend to think that we start with data distribution and once we have specified observed data, it becomes likelihood.

Reply:
I tried to update the text a little bit to make this more clear. It is generally a confusing topic. Given my comment above and this correction, let me know if it is more clear.